lib/Importer/ONNXModelLoader.cpp

```diff
@@ -780,6 +780,39 @@ llvm::Error ONNXModelLoader::loadMatMul(const ONNX_NAMESPACE::NodeProto &op,
   return llvm::Error::success();
 }
 
+llvm::Error ONNXModelLoader::loadLeakyRelu(const ONNX_NAMESPACE::NodeProto &op,
+                                           const ArgumentDictionaryTy &dict) {
+  // Input Type
+  NodeValue input;
+  ASSIGN_VALUE_OR_RETURN_ERR(input,
+                             getNodeValueOrCreateConstantByName(op.input(0)));
+  ElemKind inputType = input.getType()->getElementType();
+
+  // Only supports float types.
+  RETURN_ERR_IF_NOT((inputType == ElemKind::FloatTy) ||
+                        (inputType == ElemKind::Float16Ty),
+                    "Unsupported Type for LeakyRelu");
+
+  float alphaVal;
+  if (dict.count("alpha")) {
+    ASSIGN_VALUE_OR_RETURN_ERR(alphaVal, loadFloat(dict.at("alpha")));
+  } else {
+    // ONNX spec says default is 0.01, but doesn't explicitly say it's optional
+    // like for others. The default example just ommits alpha.
+    alphaVal = 0.01f;
```

Comment:
could just set this as the initial value on line 796

Reply:
done